spring-petclinic-microservices/spring-petclinic-api-gateway/src/main/resources/application.yml

```diff
@@ -1,5 +1,10 @@
 spring:
   cloud:
+    # With Spring Boot 2.4,
```

Comment:
Should here be Spring Cloud xxx?

Reply:
this info here is to tell why need this configuration